wof/core.py

```diff
@@ -104,6 +104,16 @@ class WOFConfig(object):
         default_east = None
 
 
+        organization = 'MYORGANIZATION'
+        address = '1234 DriveIn'
+        city = 'Seattle'
+        state = 'Washington'
+        zipcode = '98064'
+        contactname = 'John Smith'
+        contactemail = 'johnsmith@example.com'
+        phone = '555-555-555'
+        link = 'http://www.example.com/'
+
```

Comment:
Hmm, this cfg text doesn't match the screenshots you included. That's not a problem per se, but it would've been more useful if the screenshots were based directly on the sample cfg content.

Reply:
Is this default content overwritten by content on the DAO's cfg?

Reply:
@emiliom This is the default value... In case the .cfg doesn't have any value in it.

Reply:
That's what I figured. Thanks.